### PR TITLE
v5.0.0 Release

### PR DESCRIPTION
**Pull request by ashleynolan**
v5.0.0
------------------------------
*August 23, 2021*

## Main changes:
- Colour and typography variables updated to use `pie-design-tokens` repo instead of `fozzie-colour-palette`. 
- When importing v5 fozzie, consuming applications only get included variables, functions and mixins (no classes). Component styles for buttons, form-fields etc are being deprecated as we move our components over to versionable packages, but can still be included via optional mixins applications that still need this as part of the transition.
- Two optional variables can be used to include a base set of styles (similar to v4). These are `$includeBaseFramework` and `$includeMinimalFramework`. Check out the `_templates.scss` file to see what styles are included when these are set to true.

### Changed
- Eyeglass version bumped to v3.
- Made all styles and components optional (wrapped in a mixin).
- Lots of colour variables moved over as part of using the new `pie-design-tokens` repo instead of `fozzie-colour-palette`.
- Updated type-map with new typography variables for font-size and line-height.

### Removed
- `/settings/glyphs` moved into badges, as it is the only place it is used.
- `/images` as not used.
- `kickoff-grid` dependency brought into fozzie, to remove external dependency (making it easier to change if needed).
- `$font-weight-bold` was removed as it now comes straight from `pie-design-tokens`.

** Full list of changes (including all colour variable changes) can be found in beta release CHANGELOGs (see full CHANGELOG entries) **

## UI Review Checks

- [x] This PR has been checked with regard to our brand guidelines
- [x] UI Documentation has been updated (more incoming shortly)

### CODE REVIEW

**Review by DamianMullins — APPROVED** [review]
Looks good to me, and haven't noticed any issues whilst using the beta version on menu. 

One thing to mention, to future-proof us a little, could we consider removing the `$spacing` variables `x0.5` & `x1.5` as they aren't using valid syntax? Where people are using those variables they can replace it with something like `spacing() * 0.5` if they need to use the fraction values?

If not in v5 possibly in v6?

**Comment by ashleynolan** [comment]
@DamianMullins Yeh – funnily enough, I think we hit this when trying to upgrade one of our dependencies in the component repo, so I think that'll happen pretty shortly 👍

**Review by kevinrodrigues — APPROVED** [review]

File: src/scss/objects/_tables.scss [hunk 1]
```diff
@@ -1,97 +1,102 @@
...
-$table-innerBorder--color     : $color-border !default;
-$table-verticalBorder--color  : $color-border--interactive !default;
+    /**
+    * Define associated Table variables
+    */
+    $table-bgColor                : $color-container-default !global; // Default background color used for all tables.
+    $table-bgColor--accent        : $color-grey-10 !global; // Background color used for `.table-striped`.
+    $table-border--color          : $color-grey-10 !global; // Border color for table and cell borders.
+    $table-border--width          : 2px !global; // Border width for table border.
+    $table-innerBorder--color     : $color-grey-40 !global;
```
Comment (dandel10n) [inline src/scss/objects/_tables.scss:25]:
```suggestion
    $table-innerBorder--color     : $color-border-strong !global;
```

File: src/scss/trumps/_utilities.scss [hunk 2]
```diff
@@ -1,409 +1,411 @@
...
-.u-lightenBg {
-    background-color: $white;
-}
-
-.u-bordered,
-%u-bordered {
-    @include bordered();
-}
+    .u-lightenBg {
+        background-color: $color-white;
```
Comment (dandel10n) [inline src/scss/trumps/_utilities.scss:26]:
```suggestion
        background-color: $color-container-default;
```

**Pull request merged**